docs/using.rst

```diff
@@ -9,34 +9,46 @@ Getting Certbot
 ===============
 
 To get specific instructions for installing Certbot on your OS, we recommend
-visiting certbot.eff.org_. If you're offline, you can find some general
-instructions `in the README / Introduction <intro.html#installation>`__
+visiting certbot.eff.org_. For general information on how to install Certbot, 
+and the difference between the ``certbot``, ``certbot-auto``, ``letsencrypt``, 
+and ``letsencrypt-auto`` commands, please refer to the 
+`README / Introduction <intro.html#installation>`__
 
 __ installation_
 .. _certbot.eff.org: https://certbot.eff.org
 
 .. _certbot-auto:
 
-The name of the certbot command
--------------------------------
-
-Many platforms now have native packages that give you a ``certbot`` or (for
-older packages) ``letsencrypt`` command you can run. On others, the
-``certbot-auto`` / ``letsencrypt-auto`` installer and wrapper script is a
-stand-in. Throughout the documentation, whenever you see references to
-``certbot`` script/binary, you should substitute in the name of the command
-that certbot.eff.org_ told you to use on your system (``certbot``,
-``letsencrypt``, or ``certbot-auto``).
+Commands
+========
 
+The Certbot client uses a number of different "commands" (also referred
+to, equivalently, as "subcommands") to request specific actions such as
```

Comment:
I just checked, and there's [one place](https://github.com/certbot/certbot/blob/master/certbot/cli.py#L668) where the term "verb" for "subcommand" has crept from our code into our docs.  We should probably fix that, or alternatively document that subcommands can also be called verbs.